interactive_engine/executor/ir/graph_proxy/src/apis/graph/mod.rs

```diff
@@ -168,7 +168,7 @@ impl QueryParams {
     }
 
     pub fn is_queryable(&self) -> bool {
-        !(self.labels.is_empty() && self.filter.is_none() && self.limit.is_none() && self.columns.is_none())
+        !(self.filter.is_none() && self.limit.is_none() && self.columns.is_none())
```

Comment:
Why remove `labels.is_empty()` here?

Reply:
I removed this function as it is not used.